.github/workflows/e2e-test.yml

```diff
@@ -16,15 +16,7 @@ on:
       - fix-release*
 
 env:
-  GreenfieldTag: v0.2.1-alpha.1
-  GreenfieldIntegrationTestBranch: sp-refactor-0.2.1-new-sdk
-  GOPRIVATE: github.com/bnb-chain
   GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
```

Comment:
Remove all `GH_ACCESS_TOKEN ` in action yml.

Reply:
fixed